app/src/test/java/nl/tudelft/b_b_w/blockchain/BlockDataTest.java

```diff
@@ -0,0 +1,84 @@
+package nl.tudelft.b_b_w.blockchain;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+/**
+ * This class tests the functions of the BlockData object.
+ */
+public class BlockDataTest {
+
+    /**
+     * The BlockData attributes
+     */
+    private BlockData blockData;
+    private BlockType blockType;
+    private int sequenceNumber;
+    private Hash previousHashChain;
+    private Hash previousHashSender;
+    private int trustValue;
+
+    /**
+     * Setting up the test object
+     */
+    @Before
+    public void setUpBlockData(){
+
+        blockType = BlockType.ADD_KEY;
+        sequenceNumber = 5;
+        previousHashChain = new Hash("ExampleHash1");
+        previousHashSender = new Hash("ExampleHash2");
+        trustValue = 1;
+
+        blockData = new BlockData(blockType, sequenceNumber, previousHashChain, previousHashSender, trustValue);
+    }
+
+    /**
+     * getBlockType() getter method testing
+     * @throws Exception
+     */
+    @Test
+    public void getBlockTypeTest() throws Exception {
```

Comment:
For all your tests, defining which exception would be thrown is better, than catching the general exception